tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -2373,6 +2373,60 @@ public function provider_checkout_session_complete_states(): array {
+
+		$this->mock_gateway->method( 'stripe_request' )
+			->with( $request_url )
+			->willReturn( $session_mock );
+
+		// The settlement lock belongs to the webhook; the redirect handler must never take it.
+		$order_helper = $this->createPartialMock(
+			WC_Stripe_Order_Helper::class,
+			[ 'lock_order_payment', 'unlock_order_payment' ]
+		);
+		$order_helper->expects( $this->never() )->method( 'lock_order_payment' );
+		WC_Stripe_Order_Helper::set_instance( $order_helper );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Order_Helper' in method 'test_process_checkout_session_redirect_does_not_lock_order_payment'. (undefined)

(StaticAccess)